From 0bd54b222776560e0829e93380c2677ccbf20f82 Mon Sep 17 00:00:00 2001
From: xzl <18256025235@163.com>
Date: Sat, 16 Nov 2019 10:49:25 +0800
Subject: [PATCH] Change Admin.java

---
 Admin.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Admin.java b/Admin.java
index 79179b5..edcebdf 100644
--- a/Admin.java
+++ b/Admin.java
@@ -2,5 +2,6 @@ public class Admin{
 	public static void main(String[] args) {
 		private List<Emp> emps;
 		private Integer Number;
+		private String aid;
 	}
 }
\ No newline at end of file
-- 
2.23.0.windows.1

